wiki/en/Tips-Tricks-More.md

```diff
@@ -80,21 +80,92 @@ Here is the script:
+
+     This would listen on MIDI channel 2 and use MIDI controller 50 to control Jamulus channel 0 fader and so on.
+
+2. The long form is a sequence of offsets and counts for various controllers:
+
+   ```
+   [MIDI channel];[control letter][offset](*[count](;...))
+   ```
+
+   This format is stricter with its parsing, requiring numbers to be numbers and letters to be known values.
+
+   * `MIDI channel` defaults to `0`, meaning "any channel".
+
+   * `control letter` defines which Jamulus Control the MIDI controller number is assigned to:
+
+     | control letter | Jamulus Control |
+     |--------:|---------|
+     | `f` | Fader |
+     | `p` | Pan   |
+     | `s` | Solo  |
+     | `m` | Mute  |
+
+   * `offset` is the base MIDI CC number for the control.
+
+   * `count` is the number of CC values for the control, defaulting to 1 (i.e. the number Jamulus channels that can be controlled).
+
+   * Two additional `control letter` values are available:
+
+     1. `o` controls Mute Myself and has a single `offset` (i.e. `count` is ignored and taken as 1).
+
+     2. `z` is an option to reserve the first CC number for a control to mean "my Jamulus channel".  For example
+
+        ```
+        --ctrlmidich "1;f0*9;z"
+        ```
+
+        would mean MIDI CC0 controls "my" Jamulus channel fader, with CC1 to CC8 for Jamulus channels 0 to 7 (so if you were channel 3, you get two MIDI CC controls - CC0 and CC4).  (Remember that this will *not* affect the sound others hear.)
+
+   An example for a Korg nanoKONTROL2 using eight sliders starting at MIDI CC 0 to control faders and eight knobs starting at MIDI CC 16 to control pan, on any MIDI channnel, for eight Jamulus channels would be
+
+   ```
+   --ctrlmidich "0;f0*8;p16*8"
+   ```
+
+   Another example:
+
+   ```
+   --ctrlmidich "1;f0*9;p16*9;s32*9;m48*9;o64;z"
+   ```
+
+   Here, Jamulus listens on MIDI channel 1 and uses nine MIDI controller numbers for each Jamulus control: one for "My Channel" plus eight for Jamulus channels 0 to 7.  MIDI CC numbers 0 (for "My Channel") to 8 control faders.  Pan controls start at CC number 16 ("my") and end at 24; Solo 32 ("my") to 40 and Mute 48 ("my") to 56; Mute Myself is enabled/disabled by CC number 64.
+
+Please note that buttons on your MIDI controller need to be set to "toggle" mode. This means that when pressed to 'turn on' a control, it must send a MIDI CC number with a value >=64, and to 'turn off' the control it must send the same CC number with a value <64. You can read your controller's manual to find out how to set this.
+
+*Note*: Jamulus does not provide feedback on the on/off state of buttons, meaning that your controller must keep track and toggle LEDs (if any) to 'on' or 'off' itself.
+
+Jamulus channel numbers are assigned by the server and displayed in the mixer when you use `--ctrlmidich`.  When you join a server, you can get a different Jamulus channel depending on who else is already there (hence using the "my channel" option, `z`).
```

Comment:
The use of the term "Jamulus channel" - is that to differentiate it from another type of channel? If so, I'm not sure if that's clear.

Reply:
MIDI has channels and Jamulus has channel but they mean different (although somewhat similar) things - check the picture:
https://cecm.indiana.edu/361/midi.html

Reply:
Oh I see, in which case it's normal to refer to the software's channel in this way.